language/move-prover/stackless-bytecode-generator/src/dataflow_analysis.rs

```diff
@@ -24,12 +24,12 @@ pub trait TransferFunctions {
     type State: AbstractDomain + Clone;
     type InstrType;
```

Comment:
We likely don't need this abstraction any more, as our cfg is now for our bytecode.

Reply:
Removed InstrType.